src/routes/contracts/contracts.controller.spec.ts

```diff
@@ -0,0 +1,140 @@
+import { faker } from '@faker-js/faker';
+import { INestApplication } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+import * as request from 'supertest';
+import { TestAppProvider } from '../../app.provider';
+import {
+  fakeConfigurationService,
+  TestConfigurationModule,
+} from '../../config/__tests__/test.configuration.module';
+import {
+  fakeCacheService,
+  TestCacheModule,
+} from '../../datasources/cache/__tests__/test.cache.module';
+import {
+  mockNetworkService,
+  TestNetworkModule,
+} from '../../datasources/network/__tests__/test.network.module';
+import { DomainModule } from '../../domain.module';
+import { chainBuilder } from '../../domain/chains/entities/__tests__/chain.builder';
+import { contractBuilder } from '../../domain/contracts/entities/__tests__/contract.builder';
+import { ValidationModule } from '../../validation/validation.module';
+import { ContractsModule } from './contracts.module';
+
+describe('Contracts controller', () => {
```

Comment:
Missing closing the app after the tests (`afterAll`)

Reply:
True 🙈, good catch, fixed in [39b407d](https://github.com/safe-global/safe-client-gateway-nest/pull/304/commits/39b407d1bfcdd00e543e0b867aff491f0cae3601)